clients/src/main/java/org/apache/kafka/clients/consumer/internals/ClassicKafkaConsumer.java

```diff
@@ -146,7 +146,7 @@ public class LegacyKafkaConsumer<K, V> implements ConsumerDelegate<K, V> {
     // to keep from repeatedly scanning subscriptions in poll(), cache the result during metadata updates
     private boolean cachedSubscriptionHasAllFetchPositions;
 
-    LegacyKafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializer, Deserializer<V> valueDeserializer) {
+    ClassicKafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializer, Deserializer<V> valueDeserializer) {
```

Comment:
we should also remove the reference to LegacyKafkaConsumer right above, on ln 140

Reply:
Good call! I'll look for more text matches.

Reply:
I _believe_ I've found all of the occurrences of `LegacyKafkaConsumer` in code, comments, etc.